project/apps/user-service/src/adapters/controllers/auth.controller.ts

```diff
@@ -22,8 +21,8 @@ export class AuthController {
 
   @MessagePattern({ cmd: 'me' })
   async me(@Payload() token: string) {
-    const { userData } = await this.authService.me(token);
-    return { userData };
+    const userData = await this.authService.me(token);
+    return { userData }; // check: necessary to wrap in object?
```

Comment:
check this

Reply:
don't think so since required output is just user data. i'll remove in my PR for user service once this is merged